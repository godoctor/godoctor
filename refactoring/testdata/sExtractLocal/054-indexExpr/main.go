package main

import "fmt"

func main() {
	p := make([]string, 1)
	p[0] = "string"
	if p[0] != "/" { // <<<<< extractLocal,8,5,8,9,newVar,pass
		fmt.Println("Hello, 世界")
	}
}
