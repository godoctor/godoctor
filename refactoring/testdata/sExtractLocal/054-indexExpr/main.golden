package main

import "fmt"

func main() {
	p := make([]string, 1)
	p[0] = "string"
	newVar := p[0]
	if newVar != "/" { // <<<<< extractLocal,8,5,8,9,newVar,pass
		fmt.Println("Hello, 世界")
	}
}
